CharlieBackend.Data/Repositories/Impl/StudentRepository.cs

```diff
@@ -57,5 +57,10 @@ public async Task<Student> GetStudentByEmailAsync(string email)
                     .FirstOrDefaultAsync(student => 
                     student.Account.Email == email && student.Account.Role == UserRole.Student);
         }
+
+        public bool IsStudentExist(long studentId)
+        {
+            return _applicationContext.Students.Any(x => x.Id == studentId);
+        }
```

Comment:
please make this method generic and move it to Repository.cs so it can be used not only for students. Also, please use async version of .Any